releasenotes/notes/renew_gpu_binaries-2cf3eba0853b8407.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    Changed supported CUDA version for GPU simulator to 11.2 or higher.
+    Now GPU binaries are including cuQuantum support. There are separate
+    binaries for CUDA 11 and 12. qiskit-aer-gpu-cu11 for CUDA 11 and
+    qiskit-aer-gpu-cu12 for CUDA 12.
```

Comment:
I think it's critical we document the upgrade path for existing users of `qiskit-aer-gpu`  here. Users already have enough difficulty navigating the gpu support in aer for whatever reason, so I think we need to explicitly document in this release note how you go from `qiskit-aer-gpu` to `qiskit-aer-gpu-cu11`  or `qiskit-aer-gpu-cu12` and also put this in the README too.